README.md

```diff
@@ -85,3 +85,11 @@ img %>%
   image_scale("1000x200") %>% 
   image_write("content/articles/googledrive-initial-release-wd.jpg")
 ```
+
+### Inline images
+
+Inline images can be added to articles by placing them in the `/static/images/` 
+directory. By current convention, these go in a subdirectory for each post.
+Images are added to the markdown using html, e.g. 
+`<img src="/static/images/subdir/image-name.jpg" />`.
```

Comment:
I think you can still use markdown, the key is the full url

Reply:
Just gave this a whirl, and I think it's not working in markdown because of the config:
https://github.com/tidyverse/tidyverse.org/blob/cd7dc18c933738248775229be46127ec69eae591/config.toml#L2

Reply:
It worked for me in the testthat 2.0.0 post. Can you please double check?

Reply:
You're right (shocking, I know)— the problem was that I was using `"/images/blah/blah.gif"` with the quotation marks! 🙈